libnopegl/src/backends/gl/glfeatures_data.h

```diff
@@ -171,5 +173,10 @@ static const struct glfeature {
         .extensions     = (const char*[]){"GL_ARB_buffer_storage", NULL},
         .funcs_offsets  = (const size_t[]){OFFSET(BufferStorage),
                                            -1}
-    },
+    }, {
+        .name           = "texture_norm16",
+        .flag           = NGLI_FEATURE_GL_TEXTURE_NORM16,
+        .version        = 300,
+        .es_extensions  = (const char*[]){"EXT_texture_norm16", NULL},
+    }
```

Comment:
can you keep the trailing `,` to reduce the diff?

Reply:
Done.